Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -812,31 +892,23 @@ - (NSString *)cellReuseIdentifierForCellData:(MXKCellData*)cellData
 - (void)dataSource:(MXKDataSource *)dataSource didRecognizeAction:(NSString *)actionIdentifier inCell:(id<MXKCellRendering>)cell userInfo:(NSDictionary *)userInfo
 {
     // Handle here user actions on recents for Riot app
-    if ([actionIdentifier isEqualToString:kInviteRecentTableViewCellPreviewButtonPressed])
+    if ([actionIdentifier isEqualToString:kInviteRecentTableViewCellAcceptButtonPressed])
```

Comment:
We still want to preview rooms. We want to directly accept only DM.

Reply:
It's handled like Android you need to tap background cell. Preview action in `[RecentsViewController tableView:didSelectRowAtIndexPath:]`